src/pages/Home/index.tsx

```diff
@@ -7,9 +7,14 @@ import SearchBar from 'components/SearchBar';
 import CustomizeModal from 'components/Modal';
 import BobaModalForm from './CustomizeBobaModalBody';
 import Button from 'react-bootstrap/Button';
+import Spinner from 'react-bootstrap/Spinner';
+import useApi from 'hooks/API';
 
 const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
-  const [filteredBobaList, setFilteredBobaList] = useState<BobaProps[]>(bobaList);
+
+  const { data, loading } = useApi(process.env.REACT_APP_BOBA_FETCH as string);
```

Comment:
Don't know how your `env.REACT_APP_BOBA_FETCH` looks like
I assume it's like `localhost:8000/bobalist`

Can you make it more generic like 
```const REACT_APP_BACKEND_API = "http://localhost:8000:"```
And reconstruct with `${process.env.REACT_APP_BOBA_FETCH}/bobalist` so that you can re-use one variable for many other endpoints

Reply:
inside the .env ? like you can do const inside there ?

Reply:
i am just using as example.
Is `REACT_APP_BACKEND_API = "http://localhost:8000"` or `REACT_APP_BACKEND_API = "http://localhost:8000/bobalist"
if it's `REACT_APP_BACKEND_API = "http://localhost:8000", then you can just append `/bobalist` after it's imported. making it extensible for use with other endpoint

Reply:
got it, updated on